src/modules/iframe/components/activateMapButton/ActivateMapButton.js

```diff
@@ -78,8 +79,14 @@ export class ActivateMapButton extends MvuElement {
 	}
 
 	_isVisible() {
-		const queryParams = new URLSearchParams(this._environmentService.getWindow().location.search);
-		return this._environmentService.isEmbedded() && queryParams.get(QueryParameters.T_DISABLE_INITIAL_UI_HINTS) !== 'true';
+		const queryParams = this._environmentService.getUrlParams();
+		const showActivateMapButton = () => {
+			// check if we have a query parameter overdrive the iframe activateMapButton
+			const iframeComponents = queryParams.get(QueryParameters.IFRAME_COMPONENTS);
+			return iframeComponents ? !iframeComponents.split(',').includes(IFrameComponents.ACTIVATE_MAP_BUTTON) : true;
```

Comment:
Woulld suggest 
```
return iframeComponents ? iframeComponents.split(',').includes(IFrameComponents.ACTIVATE_MAP_BUTTON) : true;
```
Which means

1. when no IFRAME_COMPONENTS  query parameter is present, we show the ActivateMapButton
2. when we have an  IFRAME_COMPONENTS  query parameter  we check if the